### PR TITLE
CLN: Fix and lint for E124 closing bracket does not match visual indentation

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: tools/generate_formula_api.py [hunk 1]
```diff
@@ -43,16 +43,15 @@ def iter_subclasses(cls, _seen=None, template_classes=[]):
 def write_formula_api(directory):
     template_classes = ['DiscreteModel', 'BinaryModel', 'MultinomialModel',
                         'OrderedModel', 'CountModel',
-                      'LikelihoodModel', 'GenericLikelihoodModel',
-                      'TimeSeriesModel',
-                      # this class should really be deleted
-                      'ARIMAProcess',
-                      # these need some more work, so don't expose them
-                      'ARIMA', 'VAR', 'SVAR', 'AR', 'NBin', 'NbReg', 'ARMA',
-                      ]
```
Comment (josef-pkt) [inline tools/generate_formula_api.py]: some other tools recommend this to minimize line changes when adding additional terms

File: statsmodels/genmod/tests/test_glm_weights.py [hunk 2]
```diff
@@ -251,8 +251,8 @@ def setup_class(cls):
         # in stata and here will make this less sketchy.
         cls.corr_fact = np.sqrt((wsum - 1.) / wsum) * 0.98518473599905609
         cls.res1 = GLM(cpunish_data.endog, cpunish_data.exog,
-                        family=sm.families.Poisson(), var_weights=aweights
-                        ).fit(cov_type='HC0') #, cov_kwds={'use_correction':False})
+                       family=sm.families.Poisson(), var_weights=aweights
+                       ).fit(cov_type='HC0') #, cov_kwds={'use_correction':False})
```
Comment (josef-pkt) [inline statsmodels/genmod/tests/test_glm_weights.py]: this type I usually rewrite by defining 
mod = ...
res = mod.fit(...

File: statsmodels/examples/ex_kernel_regression3.py [hunk 3]
```diff
@@ -35,9 +35,8 @@
                              #exog=[np.column_stack((x, x**2))], reg_type='lc',
                              exog=[x, x2], reg_type='ll',
                              var_type='cc', bw='aic', #'cv_ls', #[0.23, 434697.22], #'cv_ls',
-                             censor_val=c_val[:,None],
+                             censor_val=c_val[:,None])
                              #defaults=nparam.EstimatorSettings(efficient=True)
-                             )
```
Comment (josef-pkt) [inline statsmodels/examples/ex_kernel_regression3.py]: this kind of parenthesis are intentional so it is easy to add keywords

File: statsmodels/iolib/summary.py [hunk 4]
```diff
@@ -175,15 +172,13 @@ def summary(self, yname=None, xname=None, title=0, alpha=.05,
...
-            }
+            'GLM' : self.t()}
     prob_stats = {'OLS' : self.pvalues,
                  'GLS' : self.pvalues,
                  'GLSAR' : self.pvalues,
                  'WLS' : self.pvalues,
                  'RLM' : self.pvalues,
-                 'GLM' : self.pvalues
-                }
+                 'GLM' : self.pvalues}
```
Comment (josef-pkt) [inline statsmodels/iolib/summary.py]: here also the bracket on a new line is intentional

**Comment by josef-pkt** [comment]
There are many cases where parenthesis or brackets were chosen intentionally in the current pattern and should not be changed

File: tools/generate_formula_api.py (hunk 1 above)
Comment (jbrockmendel) [reply tools/generate_formula_api.py]: I'll see if I can find a way to make both tools happy.

File: statsmodels/genmod/tests/test_glm_weights.py (hunk 2 above)
Comment (jbrockmendel) [reply statsmodels/genmod/tests/test_glm_weights.py]: I like it!  Will update.

**Comment by jbrockmendel** [comment]
Updated with requested edits (and rebased)

**Comment by jbrockmendel** [comment]
@josef-pkt I think all the cases you were concerned about have been updated.  It’d be nice to get this out of the way.

**Comment by jbrockmendel** [comment]
@josef-pkt ping; can we handle this without another round of CI?  The travis timeout is unrelated.

**Comment by bashtage** [comment]
Rebase and it is in.

**Pull request merged**